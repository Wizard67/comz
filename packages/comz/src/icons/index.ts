export { default as accessibleSvg } from './accessible.vue'
export { default as activitySvg } from './activity.vue'
export { default as adSvg } from './ad.vue'
export { default as adjustmentsSvg } from './adjustments.vue'
export { default as alarmSvg } from './alarm.vue'
export { default as alertCircleSvg } from './alert-circle.vue'
export { default as alertTriangleSvg } from './alert-triangle.vue'
export { default as alienSvg } from './alien.vue'
export { default as alignCenterSvg } from './align-center.vue'
export { default as alignJustifiedSvg } from './align-justified.vue'
export { default as alignLeftSvg } from './align-left.vue'
export { default as alignRightSvg } from './align-right.vue'
export { default as ambulanceSvg } from './ambulance.vue'
export { default as anchorSvg } from './anchor.vue'
export { default as apertureSvg } from './aperture.vue'
export { default as appsSvg } from './apps.vue'
export { default as archiveSvg } from './archive.vue'
export { default as arrowBackUpSvg } from './arrow-back-up.vue'
export { default as arrowBackSvg } from './arrow-back.vue'
export { default as arrowBarDownSvg } from './arrow-bar-down.vue'
export { default as arrowBarLeftSvg } from './arrow-bar-left.vue'
export { default as arrowBarRightSvg } from './arrow-bar-right.vue'
export { default as arrowBarUpSvg } from './arrow-bar-up.vue'
export { default as arrowDownCircleSvg } from './arrow-down-circle.vue'
export { default as arrowDownLeftCircleSvg } from './arrow-down-left-circle.vue'
export { default as arrowDownLeftSvg } from './arrow-down-left.vue'
export { default as arrowDownRightCircleSvg } from './arrow-down-right-circle.vue'
export { default as arrowDownRightSvg } from './arrow-down-right.vue'
export { default as arrowDownSvg } from './arrow-down.vue'
export { default as arrowForwardUpSvg } from './arrow-forward-up.vue'
export { default as arrowForwardSvg } from './arrow-forward.vue'
export { default as arrowLeftCircleSvg } from './arrow-left-circle.vue'
export { default as arrowLeftSvg } from './arrow-left.vue'
export { default as arrowNarrowDownSvg } from './arrow-narrow-down.vue'
export { default as arrowNarrowLeftSvg } from './arrow-narrow-left.vue'
export { default as arrowNarrowRightSvg } from './arrow-narrow-right.vue'
export { default as arrowNarrowUpSvg } from './arrow-narrow-up.vue'
export { default as arrowRightCircleSvg } from './arrow-right-circle.vue'
export { default as arrowRightSvg } from './arrow-right.vue'
export { default as arrowUpCircleSvg } from './arrow-up-circle.vue'
export { default as arrowUpLeftCircleSvg } from './arrow-up-left-circle.vue'
export { default as arrowUpLeftSvg } from './arrow-up-left.vue'
export { default as arrowUpRightCircleSvg } from './arrow-up-right-circle.vue'
export { default as arrowUpRightSvg } from './arrow-up-right.vue'
export { default as arrowUpSvg } from './arrow-up.vue'
export { default as arrowsDiagonal2Svg } from './arrows-diagonal-2.vue'
export { default as arrowsDiagonalSvg } from './arrows-diagonal.vue'
export { default as arrowsHorizontalSvg } from './arrows-horizontal.vue'
export { default as arrowsMaximizeSvg } from './arrows-maximize.vue'
export { default as arrowsMinimizeSvg } from './arrows-minimize.vue'
export { default as arrowsSortSvg } from './arrows-sort.vue'
export { default as arrowsVerticalSvg } from './arrows-vertical.vue'
export { default as artboardSvg } from './artboard.vue'
export { default as atSvg } from './at.vue'
export { default as atom2Svg } from './atom-2.vue'
export { default as atomSvg } from './atom.vue'
export { default as awardSvg } from './award.vue'
export { default as backspaceSvg } from './backspace.vue'
export { default as ballBasketballSvg } from './ball-basketball.vue'
export { default as ballBowlingSvg } from './ball-bowling.vue'
export { default as ballTennisSvg } from './ball-tennis.vue'
export { default as ballVolleyballSvg } from './ball-volleyball.vue'
export { default as banSvg } from './ban.vue'
export { default as bandageSvg } from './bandage.vue'
export { default as barcodeSvg } from './barcode.vue'
export { default as basketSvg } from './basket.vue'
export { default as battery1Svg } from './battery-1.vue'
export { default as battery2Svg } from './battery-2.vue'
export { default as battery3Svg } from './battery-3.vue'
export { default as battery4Svg } from './battery-4.vue'
export { default as batteryChargingSvg } from './battery-charging.vue'
export { default as batterySvg } from './battery.vue'
export { default as bedSvg } from './bed.vue'
export { default as bellSvg } from './bell.vue'
export { default as bikeSvg } from './bike.vue'
export { default as bluetoothSvg } from './bluetooth.vue'
export { default as boldSvg } from './bold.vue'
export { default as boltSvg } from './bolt.vue'
export { default as bookSvg } from './book.vue'
export { default as bookmarkSvg } from './bookmark.vue'
export { default as borderAllSvg } from './border-all.vue'
export { default as borderBottomSvg } from './border-bottom.vue'
export { default as borderHorizontalSvg } from './border-horizontal.vue'
export { default as borderInnerSvg } from './border-inner.vue'
export { default as borderLeftSvg } from './border-left.vue'
export { default as borderNoneSvg } from './border-none.vue'
export { default as borderOuterSvg } from './border-outer.vue'
export { default as borderRadiusSvg } from './border-radius.vue'
export { default as borderRightSvg } from './border-right.vue'
export { default as borderTopSvg } from './border-top.vue'
export { default as borderVerticalSvg } from './border-vertical.vue'
export { default as boxSvg } from './box.vue'
export { default as bracesSvg } from './braces.vue'
export { default as bracketsSvg } from './brackets.vue'
export { default as brandAndroidSvg } from './brand-android.vue'
export { default as brandAppleSvg } from './brand-apple.vue'
export { default as brandChromeSvg } from './brand-chrome.vue'
export { default as brandDribbbleSvg } from './brand-dribbble.vue'
export { default as brandFacebookSvg } from './brand-facebook.vue'
export { default as brandFramerSvg } from './brand-framer.vue'
export { default as brandGithubSvg } from './brand-github.vue'
export { default as brandGitlabSvg } from './brand-gitlab.vue'
export { default as brandGoogleDriveSvg } from './brand-google-drive.vue'
export { default as brandGoogleSvg } from './brand-google.vue'
export { default as brandInstagramSvg } from './brand-instagram.vue'
export { default as brandOperaSvg } from './brand-opera.vue'
export { default as brandPaypalSvg } from './brand-paypal.vue'
export { default as brandSafariSvg } from './brand-safari.vue'
export { default as brandSketchSvg } from './brand-sketch.vue'
export { default as brandSnapchatSvg } from './brand-snapchat.vue'
export { default as brandTelegramSvg } from './brand-telegram.vue'
export { default as brandTwitterSvg } from './brand-twitter.vue'
export { default as briefcaseSvg } from './briefcase.vue'
export { default as brightnessDownSvg } from './brightness-down.vue'
export { default as brightnessUpSvg } from './brightness-up.vue'
export { default as brightnessSvg } from './brightness.vue'
export { default as browserSvg } from './browser.vue'
export { default as brushSvg } from './brush.vue'
export { default as bucketSvg } from './bucket.vue'
export { default as bugSvg } from './bug.vue'
export { default as buildingArchSvg } from './building-arch.vue'
export { default as buildingBankSvg } from './building-bank.vue'
export { default as buildingBridge2Svg } from './building-bridge-2.vue'
export { default as buildingBridgeSvg } from './building-bridge.vue'
export { default as buildingChurchSvg } from './building-church.vue'
export { default as buildingCommunitySvg } from './building-community.vue'
export { default as buildingHospitalSvg } from './building-hospital.vue'
export { default as buildingPavilonSvg } from './building-pavilon.vue'
export { default as buildingSkyscrappersSvg } from './building-skyscrappers.vue'
export { default as buildingStoreSvg } from './building-store.vue'
export { default as buildingWarehouseSvg } from './building-warehouse.vue'
export { default as buildingSvg } from './building.vue'
export { default as bulbOffSvg } from './bulb-off.vue'
export { default as bulbSvg } from './bulb.vue'
export { default as busSvg } from './bus.vue'
export { default as calculatorSvg } from './calculator.vue'
export { default as calendarEventSvg } from './calendar-event.vue'
export { default as calendarMinusSvg } from './calendar-minus.vue'
export { default as calendarPlusSvg } from './calendar-plus.vue'
export { default as calendarSvg } from './calendar.vue'
export { default as cameraSvg } from './camera.vue'
export { default as carSvg } from './car.vue'
export { default as caretDownSvg } from './caret-down.vue'
export { default as caretLeftSvg } from './caret-left.vue'
export { default as caretRightSvg } from './caret-right.vue'
export { default as caretUpSvg } from './caret-up.vue'
export { default as cashSvg } from './cash.vue'
export { default as castSvg } from './cast.vue'
export { default as chartAreaLineSvg } from './chart-area-line.vue'
export { default as chartAreaSvg } from './chart-area.vue'
export { default as chartBarSvg } from './chart-bar.vue'
export { default as chartCandleSvg } from './chart-candle.vue'
export { default as chartDonutSvg } from './chart-donut.vue'
export { default as chartLineSvg } from './chart-line.vue'
export { default as chartPieSvg } from './chart-pie.vue'
export { default as checkSvg } from './check.vue'
export { default as checkboxSvg } from './checkbox.vue'
export { default as checksSvg } from './checks.vue'
export { default as chevronDownSvg } from './chevron-down.vue'
export { default as chevronLeftSvg } from './chevron-left.vue'
export { default as chevronRightSvg } from './chevron-right.vue'
export { default as chevronUpSvg } from './chevron-up.vue'
export { default as chevronsDownSvg } from './chevrons-down.vue'
export { default as chevronsLeftSvg } from './chevrons-left.vue'
export { default as chevronsRightSvg } from './chevrons-right.vue'
export { default as chevronsUpSvg } from './chevrons-up.vue'
export { default as circleCheckSvg } from './circle-check.vue'
export { default as circleMinusSvg } from './circle-minus.vue'
export { default as circlePlusSvg } from './circle-plus.vue'
export { default as circleXSvg } from './circle-x.vue'
export { default as circleSvg } from './circle.vue'
export { default as clearFormattingSvg } from './clear-formatting.vue'
export { default as clickSvg } from './click.vue'
export { default as clipboardCheckSvg } from './clipboard-check.vue'
export { default as clipboardListSvg } from './clipboard-list.vue'
export { default as clipboardXSvg } from './clipboard-x.vue'
export { default as clipboardSvg } from './clipboard.vue'
export { default as clockSvg } from './clock.vue'
export { default as cloudDownloadSvg } from './cloud-download.vue'
export { default as cloudRainSvg } from './cloud-rain.vue'
export { default as cloudSnowSvg } from './cloud-snow.vue'
export { default as cloudStormSvg } from './cloud-storm.vue'
export { default as cloudUploadSvg } from './cloud-upload.vue'
export { default as cloudSvg } from './cloud.vue'
export { default as codeSvg } from './code.vue'
export { default as coinSvg } from './coin.vue'
export { default as colorPickerSvg } from './color-picker.vue'
export { default as colorSwatchSvg } from './color-swatch.vue'
export { default as columnsSvg } from './columns.vue'
export { default as commandSvg } from './command.vue'
export { default as compassSvg } from './compass.vue'
export { default as copySvg } from './copy.vue'
export { default as copyrightSvg } from './copyright.vue'
export { default as cornerDownLeftSvg } from './corner-down-left.vue'
export { default as cornerDownRightSvg } from './corner-down-right.vue'
export { default as cornerLeftDownSvg } from './corner-left-down.vue'
export { default as cornerLeftUpSvg } from './corner-left-up.vue'
export { default as cornerRightDownSvg } from './corner-right-down.vue'
export { default as cornerRightUpSvg } from './corner-right-up.vue'
export { default as cornerUpLeftSvg } from './corner-up-left.vue'
export { default as cornerUpRightSvg } from './corner-up-right.vue'
export { default as creditCardSvg } from './credit-card.vue'
export { default as cropSvg } from './crop.vue'
export { default as currencyBitcoinSvg } from './currency-bitcoin.vue'
export { default as currencyDollarSvg } from './currency-dollar.vue'
export { default as currencyEuroSvg } from './currency-euro.vue'
export { default as currencyPoundSvg } from './currency-pound.vue'
export { default as currencyRupeeSvg } from './currency-rupee.vue'
export { default as currencyYenSvg } from './currency-yen.vue'
export { default as cutSvg } from './cut.vue'
export { default as dashboardSvg } from './dashboard.vue'
export { default as databaseSvg } from './database.vue'
export { default as deviceDesktopSvg } from './device-desktop.vue'
export { default as deviceFloppySvg } from './device-floppy.vue'
export { default as deviceGamepadSvg } from './device-gamepad.vue'
export { default as deviceLaptopSvg } from './device-laptop.vue'
export { default as deviceMobileVibrationSvg } from './device-mobile-vibration.vue'
export { default as deviceMobileSvg } from './device-mobile.vue'
export { default as deviceSpeakerSvg } from './device-speaker.vue'
export { default as deviceTabletSvg } from './device-tablet.vue'
export { default as deviceTvSvg } from './device-tv.vue'
export { default as deviceWatchSvg } from './device-watch.vue'
export { default as devicesSvg } from './devices.vue'
export { default as diamondSvg } from './diamond.vue'
export { default as diceSvg } from './dice.vue'
export { default as directionHorizontalSvg } from './direction-horizontal.vue'
export { default as directionSvg } from './direction.vue'
export { default as directionsSvg } from './directions.vue'
export { default as disabled2Svg } from './disabled-2.vue'
export { default as disabledSvg } from './disabled.vue'
export { default as discSvg } from './disc.vue'
export { default as discountSvg } from './discount.vue'
export { default as dotsCircleHorizontalSvg } from './dots-circle-horizontal.vue'
export { default as dotsDiagonal2Svg } from './dots-diagonal-2.vue'
export { default as dotsDiagonalSvg } from './dots-diagonal.vue'
export { default as dotsVerticalSvg } from './dots-vertical.vue'
export { default as dotsSvg } from './dots.vue'
export { default as downloadSvg } from './download.vue'
export { default as dragDrop2Svg } from './drag-drop-2.vue'
export { default as dragDropSvg } from './drag-drop.vue'
export { default as dropletSvg } from './droplet.vue'
export { default as earSvg } from './ear.vue'
export { default as editSvg } from './edit.vue'
export { default as eggSvg } from './egg.vue'
export { default as emphasisSvg } from './emphasis.vue'
export { default as eraserSvg } from './eraser.vue'
export { default as exchangeSvg } from './exchange.vue'
export { default as exposureSvg } from './exposure.vue'
export { default as externalLinkSvg } from './external-link.vue'
export { default as eyeSvg } from './eye.vue'
export { default as faceIdSvg } from './face-id.vue'
export { default as fileCheckSvg } from './file-check.vue'
export { default as fileCodeSvg } from './file-code.vue'
export { default as fileDownloadSvg } from './file-download.vue'
export { default as fileHorizontalSvg } from './file-horizontal.vue'
export { default as fileInvoiceSvg } from './file-invoice.vue'
export { default as fileMinusSvg } from './file-minus.vue'
export { default as fileMusicSvg } from './file-music.vue'
export { default as filePlusSvg } from './file-plus.vue'
export { default as fileShredderSvg } from './file-shredder.vue'
export { default as fileTextSvg } from './file-text.vue'
export { default as fileXSvg } from './file-x.vue'
export { default as fileSvg } from './file.vue'
export { default as filterSvg } from './filter.vue'
export { default as fingerprintSvg } from './fingerprint.vue'
export { default as firetruckSvg } from './firetruck.vue'
export { default as flagSvg } from './flag.vue'
export { default as flameSvg } from './flame.vue'
export { default as flaskSvg } from './flask.vue'
export { default as flipHorizontalSvg } from './flip-horizontal.vue'
export { default as flipVerticalSvg } from './flip-vertical.vue'
export { default as floatCenterSvg } from './float-center.vue'
export { default as floatLeftSvg } from './float-left.vue'
export { default as floatRightSvg } from './float-right.vue'
export { default as floppyDiskSvg } from './floppy-disk.vue'
export { default as focus2Svg } from './focus-2.vue'
export { default as focusSvg } from './focus.vue'
export { default as folderMinusSvg } from './folder-minus.vue'
export { default as folderPlusSvg } from './folder-plus.vue'
export { default as folderXSvg } from './folder-x.vue'
export { default as folderSvg } from './folder.vue'
export { default as foldersSvg } from './folders.vue'
export { default as forbid2Svg } from './forbid-2.vue'
export { default as forbidSvg } from './forbid.vue'
export { default as forkliftSvg } from './forklift.vue'
export { default as frameSvg } from './frame.vue'
export { default as friendsSvg } from './friends.vue'
export { default as gaugeSvg } from './gauge.vue'
export { default as ghostSvg } from './ghost.vue'
export { default as giftSvg } from './gift.vue'
export { default as gitBranchSvg } from './git-branch.vue'
export { default as gitCommitSvg } from './git-commit.vue'
export { default as gitCompareSvg } from './git-compare.vue'
export { default as gitForkSvg } from './git-fork.vue'
export { default as gitMergeSvg } from './git-merge.vue'
export { default as gitPullRequestSvg } from './git-pull-request.vue'
export { default as glassFullSvg } from './glass-full.vue'
export { default as glassSvg } from './glass.vue'
export { default as globeSvg } from './globe.vue'
export { default as gridDotsSvg } from './grid-dots.vue'
export { default as gridSvg } from './grid.vue'
export { default as gripHorizontalSvg } from './grip-horizontal.vue'
export { default as gripVerticalSvg } from './grip-vertical.vue'
export { default as handMiddleFingerSvg } from './hand-middle-finger.vue'
export { default as handStopSvg } from './hand-stop.vue'
export { default as hashSvg } from './hash.vue'
export { default as headphonesSvg } from './headphones.vue'
export { default as headsetSvg } from './headset.vue'
export { default as heartSvg } from './heart.vue'
export { default as helpSvg } from './help.vue'
export { default as hexagonSvg } from './hexagon.vue'
export { default as historySvg } from './history.vue'
export { default as home2Svg } from './home-2.vue'
export { default as homeSvg } from './home.vue'
export { default as iceCreamSvg } from './ice-cream.vue'
export { default as idSvg } from './id.vue'
export { default as inboxSvg } from './inbox.vue'
export { default as indentDecreaseSvg } from './indent-decrease.vue'
export { default as indentIncreaseSvg } from './indent-increase.vue'
export { default as infinitySvg } from './infinity.vue'
export { default as infoCircleSvg } from './info-circle.vue'
export { default as infoSquareSvg } from './info-square.vue'
export { default as italicSvg } from './italic.vue'
export { default as keySvg } from './key.vue'
export { default as keyboardSvg } from './keyboard.vue'
export { default as languageSvg } from './language.vue'
export { default as layersDifferenceSvg } from './layers-difference.vue'
export { default as layersIntersectSvg } from './layers-intersect.vue'
export { default as layersSubtractSvg } from './layers-subtract.vue'
export { default as layersUnionSvg } from './layers-union.vue'
export { default as layout2Svg } from './layout-2.vue'
export { default as layoutAlignBottomSvg } from './layout-align-bottom.vue'
export { default as layoutAlignCenterSvg } from './layout-align-center.vue'
export { default as layoutAlignLeftSvg } from './layout-align-left.vue'
export { default as layoutAlignMiddleSvg } from './layout-align-middle.vue'
export { default as layoutAlignRightSvg } from './layout-align-right.vue'
export { default as layoutAlignTopSvg } from './layout-align-top.vue'
export { default as layoutBottombarSvg } from './layout-bottombar.vue'
export { default as layoutCardsSvg } from './layout-cards.vue'
export { default as layoutColumnsSvg } from './layout-columns.vue'
export { default as layoutDistributeHorizontalSvg } from './layout-distribute-horizontal.vue'
export { default as layoutDistributeVerticalSvg } from './layout-distribute-vertical.vue'
export { default as layoutListSvg } from './layout-list.vue'
export { default as layoutNavbarSvg } from './layout-navbar.vue'
export { default as layoutRowsSvg } from './layout-rows.vue'
export { default as layoutSidebarRightSvg } from './layout-sidebar-right.vue'
export { default as layoutSidebarSvg } from './layout-sidebar.vue'
export { default as layoutSvg } from './layout.vue'
export { default as legoSvg } from './lego.vue'
export { default as letterCaseSvg } from './letter-case.vue'
export { default as licenseSvg } from './license.vue'
export { default as lifebuoySvg } from './lifebuoy.vue'
export { default as lineHeightSvg } from './line-height.vue'
export { default as linkSvg } from './link.vue'
export { default as listCheckSvg } from './list-check.vue'
export { default as listSvg } from './list.vue'
export { default as livePhotoSvg } from './live-photo.vue'
export { default as locationSvg } from './location.vue'
export { default as lockOpenSvg } from './lock-open.vue'
export { default as lockSvg } from './lock.vue'
export { default as loginSvg } from './login.vue'
export { default as logoutSvg } from './logout.vue'
export { default as magnetSvg } from './magnet.vue'
export { default as mailOpenedSvg } from './mail-opened.vue'
export { default as mailSvg } from './mail.vue'
export { default as manSvg } from './man.vue'
export { default as map2Svg } from './map-2.vue'
export { default as mapPinSvg } from './map-pin.vue'
export { default as mapSvg } from './map.vue'
export { default as mathSvg } from './math.vue'
export { default as maximizeSvg } from './maximize.vue'
export { default as medicalCrossSvg } from './medical-cross.vue'
export { default as menuSvg } from './menu.vue'
export { default as message2Svg } from './message-2.vue'
export { default as messageCircleSvg } from './message-circle.vue'
export { default as messageDotsSvg } from './message-dots.vue'
export { default as messageSvg } from './message.vue'
export { default as messagesSvg } from './messages.vue'
export { default as microphoneSvg } from './microphone.vue'
export { default as minimizeSvg } from './minimize.vue'
export { default as minusSvg } from './minus.vue'
export { default as mistSvg } from './mist.vue'
export { default as moodConfuzedSvg } from './mood-confuzed.vue'
export { default as moodHappySvg } from './mood-happy.vue'
export { default as moodKidSvg } from './mood-kid.vue'
export { default as moodNeutralSvg } from './mood-neutral.vue'
export { default as moodSadSvg } from './mood-sad.vue'
export { default as moodSmileSvg } from './mood-smile.vue'
export { default as moodSuprisedSvg } from './mood-suprised.vue'
export { default as moodTongueSvg } from './mood-tongue.vue'
export { default as moonSvg } from './moon.vue'
export { default as mouseSvg } from './mouse.vue'
export { default as movieSvg } from './movie.vue'
export { default as mugSvg } from './mug.vue'
export { default as musicSvg } from './music.vue'
export { default as newSectionSvg } from './new-section.vue'
export { default as newsSvg } from './news.vue'
export { default as noteSvg } from './note.vue'
export { default as notebookSvg } from './notebook.vue'
export { default as notesSvg } from './notes.vue'
export { default as notificationSvg } from './notification.vue'
export { default as omegaSvg } from './omega.vue'
export { default as outletSvg } from './outlet.vue'
export { default as packageSvg } from './package.vue'
export { default as paintSvg } from './paint.vue'
export { default as paletteSvg } from './palette.vue'
export { default as paperclipSvg } from './paperclip.vue'
export { default as parenthesesSvg } from './parentheses.vue'
export { default as parkingSvg } from './parking.vue'
export { default as pencilSvg } from './pencil.vue'
export { default as phoneCallSvg } from './phone-call.vue'
export { default as phoneCheckSvg } from './phone-check.vue'
export { default as phoneIncomingSvg } from './phone-incoming.vue'
export { default as phoneOutgoingSvg } from './phone-outgoing.vue'
export { default as phonePauseSvg } from './phone-pause.vue'
export { default as phonePlusSvg } from './phone-plus.vue'
export { default as phoneXSvg } from './phone-x.vue'
export { default as phoneSvg } from './phone.vue'
export { default as photoSvg } from './photo.vue'
export { default as pillowSvg } from './pillow.vue'
export { default as planeArrivalSvg } from './plane-arrival.vue'
export { default as planeDepartureSvg } from './plane-departure.vue'
export { default as planeSvg } from './plane.vue'
export { default as planetSvg } from './planet.vue'
export { default as plugSvg } from './plug.vue'
export { default as plusSvg } from './plus.vue'
export { default as pointSvg } from './point.vue'
export { default as powerSvg } from './power.vue'
export { default as presentationSvg } from './presentation.vue'
export { default as printerSvg } from './printer.vue'
export { default as promptSvg } from './prompt.vue'
export { default as puzzleSvg } from './puzzle.vue'
export { default as qrcodeSvg } from './qrcode.vue'
export { default as recordMailSvg } from './record-mail.vue'
export { default as recycleSvg } from './recycle.vue'
export { default as refreshSvg } from './refresh.vue'
export { default as registeredSvg } from './registered.vue'
export { default as repeatOnceSvg } from './repeat-once.vue'
export { default as repeatSvg } from './repeat.vue'
export { default as replaceSvg } from './replace.vue'
export { default as rotate2Svg } from './rotate-2.vue'
export { default as rotateClockwise2Svg } from './rotate-clockwise-2.vue'
export { default as rotateClockwiseSvg } from './rotate-clockwise.vue'
export { default as rotateRectangleSvg } from './rotate-rectangle.vue'
export { default as rotateSvg } from './rotate.vue'
export { default as routeSvg } from './route.vue'
export { default as routerSvg } from './router.vue'
export { default as rssSvg } from './rss.vue'
export { default as rulerSvg } from './ruler.vue'
export { default as scaleSvg } from './scale.vue'
export { default as scanSvg } from './scan.vue'
export { default as scissorsSvg } from './scissors.vue'
export { default as searchSvg } from './search.vue'
export { default as selectorSvg } from './selector.vue'
export { default as sendSvg } from './send.vue'
export { default as separatorSvg } from './separator.vue'
export { default as serverSvg } from './server.vue'
export { default as servicemarkSvg } from './servicemark.vue'
export { default as settingsSvg } from './settings.vue'
export { default as shapeSvg } from './shape.vue'
export { default as shareSvg } from './share.vue'
export { default as shieldCheckSvg } from './shield-check.vue'
export { default as shieldXSvg } from './shield-x.vue'
export { default as shieldSvg } from './shield.vue'
export { default as shirtSvg } from './shirt.vue'
export { default as shoppingCartSvg } from './shopping-cart.vue'
export { default as sitemapSvg } from './sitemap.vue'
export { default as sliceSvg } from './slice.vue'
export { default as slideshowSvg } from './slideshow.vue'
export { default as snowflakeSvg } from './snowflake.vue'
export { default as socialSvg } from './social.vue'
export { default as sortAscendingSvg } from './sort-ascending.vue'
export { default as sortDescendingSvg } from './sort-descending.vue'
export { default as spaceSvg } from './space.vue'
export { default as squareCheckSvg } from './square-check.vue'
export { default as squareMinusSvg } from './square-minus.vue'
export { default as squarePlusSvg } from './square-plus.vue'
export { default as squareXSvg } from './square-x.vue'
export { default as squareSvg } from './square.vue'
export { default as stackSvg } from './stack.vue'
export { default as starSvg } from './star.vue'
export { default as stickerSvg } from './sticker.vue'
export { default as strikethroughSvg } from './strikethrough.vue'
export { default as subscriptSvg } from './subscript.vue'
export { default as sumSvg } from './sum.vue'
export { default as sunSvg } from './sun.vue'
export { default as sunsetSvg } from './sunset.vue'
export { default as sunshineSvg } from './sunshine.vue'
export { default as superscriptSvg } from './superscript.vue'
export { default as switchHorizontalSvg } from './switch-horizontal.vue'
export { default as switchVerticalSvg } from './switch-vertical.vue'
export { default as switchSvg } from './switch.vue'
export { default as tableSvg } from './table.vue'
export { default as tagSvg } from './tag.vue'
export { default as targetSvg } from './target.vue'
export { default as taxSvg } from './tax.vue'
export { default as temperatureCelsiusSvg } from './temperature-celsius.vue'
export { default as temperatureFahrenheitSvg } from './temperature-fahrenheit.vue'
export { default as temperatureMinusSvg } from './temperature-minus.vue'
export { default as temperaturePlusSvg } from './temperature-plus.vue'
export { default as temperatureSvg } from './temperature.vue'
export { default as templateSvg } from './template.vue'
export { default as terminal2Svg } from './terminal-2.vue'
export { default as terminalSvg } from './terminal.vue'
export { default as testPipeSvg } from './test-pipe.vue'
export { default as testSvg } from './test.vue'
export { default as textWrapSvg } from './text-wrap.vue'
export { default as thumbDownSvg } from './thumb-down.vue'
export { default as thumbUpSvg } from './thumb-up.vue'
export { default as ticketSvg } from './ticket.vue'
export { default as tirSvg } from './tir.vue'
export { default as toggleLeftSvg } from './toggle-left.vue'
export { default as toggleRightSvg } from './toggle-right.vue'
export { default as toolSvg } from './tool.vue'
export { default as toolsSvg } from './tools.vue'
export { default as tractorSvg } from './tractor.vue'
export { default as trademarkSvg } from './trademark.vue'
export { default as trafficConeSvg } from './traffic-cone.vue'
export { default as trashSvg } from './trash.vue'
export { default as treesSvg } from './trees.vue'
export { default as trendingDownSvg } from './trending-down.vue'
export { default as trendingUpSvg } from './trending-up.vue'
export { default as triangleSvg } from './triangle.vue'
export { default as trophySvg } from './trophy.vue'
export { default as truckSvg } from './truck.vue'
export { default as typographySvg } from './typography.vue'
export { default as umbrellaSvg } from './umbrella.vue'
export { default as underlineSvg } from './underline.vue'
export { default as unlinkSvg } from './unlink.vue'
export { default as uploadSvg } from './upload.vue'
export { default as urgentSvg } from './urgent.vue'
export { default as userCheckSvg } from './user-check.vue'
export { default as userExclamationSvg } from './user-exclamation.vue'
export { default as userMinusSvg } from './user-minus.vue'
export { default as userPlusSvg } from './user-plus.vue'
export { default as userXSvg } from './user-x.vue'
export { default as userSvg } from './user.vue'
export { default as usersSvg } from './users.vue'
export { default as viewfinderSvg } from './viewfinder.vue'
export { default as viewportNarrowSvg } from './viewport-narrow.vue'
export { default as viewportWideSvg } from './viewport-wide.vue'
export { default as virusSvg } from './virus.vue'
export { default as volume2Svg } from './volume-2.vue'
export { default as volume3Svg } from './volume-3.vue'
export { default as volumeSvg } from './volume.vue'
export { default as walletSvg } from './wallet.vue'
export { default as wandSvg } from './wand.vue'
export { default as wifi0Svg } from './wifi-0.vue'
export { default as wifi1Svg } from './wifi-1.vue'
export { default as wifi2Svg } from './wifi-2.vue'
export { default as wifiSvg } from './wifi.vue'
export { default as windSvg } from './wind.vue'
export { default as womanSvg } from './woman.vue'
export { default as worldSvg } from './world.vue'
export { default as xSvg } from './x.vue'
export { default as yinYangSvg } from './yin-yang.vue'
export { default as zoomInSvg } from './zoom-in.vue'
export { default as zoomOutSvg } from './zoom-out.vue'