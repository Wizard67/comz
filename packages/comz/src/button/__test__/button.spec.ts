import { markRaw } from 'vue'
import { mount } from '@vue/test-utils'
import Button from '../button.vue'
import { X } from '@comz/icons'

describe('Button.vue', () => {
  describe('props', () => {
    it('icon', () => {
      const wrapper = mount(Button, {
        props: {
          icon: markRaw(X)
        }
      })

      const svg = wrapper.find('svg')
      expect(svg).not.toBeUndefined()
      expect(svg.classes()).toContain('cbutton__icon')
      expect(wrapper.element).toMatchSnapshot()
    })

    it('loading', () => {
      const wrapper = mount(Button, {
        props: {
          loading: true
        }
      })
  
      expect(wrapper.classes()).toContain('cbutton--loading')
      expect(wrapper.element).toMatchSnapshot()
    })

    it('disabled', () => {
      const wrapper = mount(Button, {
        props: {
          disabled: true
        }
      })
  
      expect(wrapper.classes()).toContain('cbutton--disabled')
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe('event', () => {
    it('on-click', () => {
      const wrapper = mount(Button)
      
      wrapper.trigger('click')
      expect(wrapper.emitted()).toHaveProperty('on-click')
    })
  })

  describe('slots', () => {
    it('default', () => {
      const content = "button"

      const wrapper = mount(Button, {
        slots: {
          default: content
        }
      })

      expect(wrapper.text()).toContain(content)
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
