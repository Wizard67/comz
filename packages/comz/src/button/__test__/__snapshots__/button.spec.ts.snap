// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.vue props disabled 1`] = `
<button
  class="cbutton cbutton--disabled"
  disabled=""
  type="button"
>
  <!--v-if-->
  
  
</button>
`;

exports[`Button.vue props icon 1`] = `
<button
  class="cbutton"
  type="button"
>
  <svg
    class="cbutton__icon"
    fill="currentColor"
    height="1em"
    viewBox="0 0 16 16"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"
      fill-rule="evenodd"
    />
  </svg>
  
  
</button>
`;

exports[`Button.vue props loading 1`] = `
<button
  class="cbutton cbutton--loading"
  type="button"
>
  <!--v-if-->
  
  
</button>
`;

exports[`Button.vue slots default 1`] = `
<button
  class="cbutton"
  type="button"
>
  <!--v-if-->
  
  
  button
  
  
</button>
`;
